.gitignore

```diff
@@ -25,3 +25,6 @@ target
 # Ignore target directories
 target/
 *.*~
+
+org/wso2/extension/siddhi/gpl/execution/streamingml/clustering/clustree/a1.csv
+TryMOA.class
```

Comment:
Why are these added?

Reply:
i added some classes to test moa lib then removed. ill delete

Reply:
done